### ISSUE TITLE
Configuration and testing

### ISSUE THREAD

**Issue by gyulanemeth** [nodebp/nodebp#15]
Best practices about development/production configs without hardcoding the settings into the code.

Best practices about testing nodejs applications.

An example project would be nice. (And a workshop maybe?)

**Comment by oroce** [comment]
have you tried nconf?:)

**Comment by gyulanemeth** [comment]
Yes, I found that today :D seems to be awesome. Then some talk about proper testing would be nice. It seems to be a little bit of magic to balance and find the correct amount and way of testing.

**Comment by chmanie** [comment]
Did that already happen? I could talk about testing (unit, integration, e2e), basically practises I applied in different projects. I could also pull out a lot of examples, or create a little workshop).

**Comment by EggDice** [comment]
:+1:

**Comment by gyulanemeth** [comment]
like!
